src/test/java/org/springframework/retry/listener/RetryListenerSupportTests.java

```diff
@@ -21,6 +21,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatNoException;
 
+@SuppressWarnings("deprecation")
```

Comment:
I think it is better to rework tests in this class to deal with new code base.
I believe that this `RetryListenerSupport` will be removed in the next `2.1` version.
But that doesn't mean we will remove this test class.
So, let's see if just `new RetryListener()` is enough for unit tests in this class!

Reply:
There's already a class `RetryListenerTests`. I added the corresponding tests over there, and left this class as is and ready to be removed.